### DISCUSSION TITLE
Where did you get Kazakh dataset?

### DISCUSSION THREAD

**Discussion by kurshakuz** [ria-com/nomeroff-net#124] [Q&A] [answered]
I am currently trying to build a Kazakh plate recognition tool and your implementation helps a lot to start with! Can you tell me where did you get the data for Kazakh PL OCR? I have seen the data you hosted data on your website, but I am looking to find more. Do you have any more or can suggest where I can get it? I can even label some if such data available.

**Comment by ApelSYN** [comment, marked as answer]
All Kazakh photo content was provided by Nomeroff Net users. Numberplate photos you can be bought on this resource http://avto-nomer.ru/kz/ .
